package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/css-types-loader",
-  "version": "3.1.0",
+  "version": "4.0.0",
```

Does this PR change how the consumer of the package needs to consume it? If it's not a breaking change, then a minor version bump is more in keeping with semver.
```suggestion
  "version": "3.2.0",
```

---

@knagurski It is a breaking change. Basically the MFE using the new version will have to upgrade to typescript ^5.0.x, hence added major version bump